assets/responsibleai/environments/responsibleai-vision-ubuntu20.04-py38-cpu/context/Dockerfile

```diff
@@ -12,19 +108,19 @@ ENV PATH $AZUREML_CONDA_ENVIRONMENT_PATH/bin:$PATH
 # Create conda environment
 COPY conda_dependencies.yaml .
 RUN conda env create -p $AZUREML_CONDA_ENVIRONMENT_PATH -f conda_dependencies.yaml -q && \
-    rm conda_dependencies.yaml && \
-    conda run -p $AZUREML_CONDA_ENVIRONMENT_PATH pip cache purge && \
+    rm conda_dependencies.yaml
+RUN conda run -p $AZUREML_CONDA_ENVIRONMENT_PATH pip cache purge && \
     conda clean -a -y
 
 RUN pip install --pre 'azure-ai-ml' 'azure-storage-blob<=12.13.0'
 
 # Install azureml packages
-RUN pip install 'azureml-core=={{latest-pypi-version}}' \
-                'azureml-mlflow=={{latest-pypi-version}}' \
+RUN pip install 'azureml-core' \
+                'azureml-mlflow' \
                 'azureml-rai-utils==0.0.3'
 
-RUN pip install 'azureml-dataset-runtime=={{latest-pypi-version}}' \
```

Comment:
this worries me a bit since the env management team always recommends to pin these to {{latest-pypi-version}}

Reply:
Oh, this was actually just removed for the local validation build. It doesn't like placeholders :-) For this env definition it's fine since the build in the repo knows how to take care of it. I had the change to revert this locally, just forgot to push. Thanks for the reminder!